Terminal.Gui/ConsoleDrivers/NetDriver.cs

```diff
@@ -1157,7 +1155,7 @@ private void ProcessInput (InputResult inputEvent)
 
     #region Size and Position Handling
 
-    volatile private bool _winSizeChanging;
+    private volatile bool _winSizeChanging;
```

Comment:
This is only thread-safe on the same logical CPU. Other threads or even the same thread, if it gets shunted to another CPU, are not affected by volatile, which just puts a memory barrier around accesses to the variable.